carla-carma-integration/carla_carma_bridge/src/carla_carma_bridge/carma_carla_route

```diff
@@ -0,0 +1,146 @@
+#!/usr/bin/env python
+# Copyright (C) 2021 LEIDOS.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not
+# use this file except in compliance with the License. You may obtain a copy of
+# the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations under
+# the License.
+"""
+Call Services from CARMA: 
+    Service: /guidance/get_available_routes
+             /guidance/set_active_route
+
+Publish to CARMA :cav_msgs::Route
+    
+"""
+from asyncio.windows_events import NULL
```

Comment:
I think the types on line 23-25 should be looked at, this are pretty abnormal python types so I'm not sure where they're coming into play here. Windows API stuff and the XML packages definitely aren't needed for this, and I'm not sure what the "typing" package is.